### PR TITLE
updating maintainers and codeowners

### PR DESCRIPTION
**Pull request by eirsep**
### Description
updating maintainers and codeowners. Adds @eisep @AWSHurneyt @sbcd90.
### Issues Resolved
#127

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,12 @@
-* @lezzago @qreshi @skkosuri-amzn @bowenlan-amzn @rishabhmaurya @tlfeng @getsaurabh02
\ No newline at end of file
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
+*   @opensearch-project/common-utils
```
Comment (getsaurabh02) [inline .github/CODEOWNERS:2]: can we fix the errors here?

**Review by lezzago — APPROVED** [review]

**Review by getsaurabh02 — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,3 @@
-* @lezzago @qreshi @skkosuri-amzn @bowenlan-amzn @rishabhmaurya @tlfeng @getsaurabh02
\ No newline at end of file
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams @opensearch-project/common-utils
+@lezzago @qreshi @bowenlan-amzn @rishabhmaurya @tlfeng @getsaurabh02 @eirsep @sbcd90 @AWSHurneyt
```
Comment (peterzhuamazon) [inline .github/CODEOWNERS:2]: Just realize you remove the prefix `*` before the 1st user.
Please open another PR and add back.

Thanks.

Comment (peterzhuamazon) [reply .github/CODEOWNERS:2]: @lezzago @getsaurabh02 @eirsep ^^

Comment (peterzhuamazon) [reply .github/CODEOWNERS:2]: https://github.com/opensearch-project/common-utils/pull/440

**Pull request merged**